docs/source/contributing/review_pr_pymc_examples.md

```diff
@@ -0,0 +1,113 @@
+
+(reveiw_pr/scope)=
+## 1. Define the scope
+Before starting you need to make sure you understand the scope of the PR and
+to set the scope of your own review.
+
+There are many valid contributions that can be done to the notebooks in pymc-examples.
+They can be focused on code, styling, wording...
+pymc-examples needs to have regular updates and in many cases it doesn't matter if
+these are partial and don't update everything that can be updated in the notebook.
+
+Not all partial updates are valid nor helpful though. If doing updates
+to the code for example, most of the wording will probably need no changes,
+but there may be some sections where the explanation is specifically about the code
+used. We don't know when the next partial update will come, so PRs should merge
+working and coherent notebooks. If the code is updated to use `Potential` instead
+of `DensityDist` and the explanation mentions `DensityDist`, that specific sentence
+needs to be updated too.
+
+If you are not sure about the scope of the PR or think that what the PR is aiming for
+won't leave a coherent notebook, ask the PR author to make sure you are on the same
+page before reviewing.
+
+Once you are clear on the scope of the PR, define which subset of that (if any)
+will your review cover. A PR that aims to update everything about a notebook
+could easily have 3 or more reviewers, each covering different aspects of the
+example like Aesara usage, writing and explanation of the concepts used,
+ArviZ usage, styling with MyST+Sphinx, or structuring and scope of the notebook.
+
+Unless you plan to review everything, start your review mentioning what your
+review covers (or skips).
+
+Part of that is also making sure that the PR description links to the relevant issue(s).
+
+(review_pr/initial_pass)=
+## 2. Try to be concise and clear
+Try to be concise and to the point with your reviews.
+You'll need less time to review and the PR author will need less time to go over the review.
+Here are some examples about what this does:
+
+* Skim the notebook from top to bottom before starting your review.
+  This will prevent you from spending some time writing about missing content
+  and suggesting how to add it only to find out it is not missing but badly structured.
+* Do not proofread the text until you are sure no large rewritings will be needed.
+  What is the point of reviewing/fixing a typo when the whole paragraph is to be rewritten?
+* Include some rationale about your comments. Contributors come from a wide range
+  of places, fields and personal contexts and English might not be their (nor your) first language.
+  Adding the reasons behind a suggestion goes a long way in making sure the suggestion
+  is clear and makes it more probable that the author doesn't repeat this pattern again.
+
+and doesn't mean:
+
+* Avoiding mentioning good things from the PR or not thanking the PR author.
+* Being cryptic or writing badly
+
+## 3. Review the content and code
```

Comment:
By "content" do you mean supporting text? Might mention something about ensuring relevance and clarity of text, as well as filling in missing text where code requires some explanation.